NDB.Covid19/NDB.Covid19/ExposureNotifications/Helpers/FetchExposureKeys/FetchExposureKeysHelper.cs

```diff
@@ -106,7 +106,7 @@ private async void ResendMessageIfNeeded()
         private void SendReApproveConsentsNotificationIfNeeded()
         {
             if (ConsentsHelper.IsNotFullyOnboarded &&
-                !LocalPreferencesHelper.TermsNotificationWasShown)
+                (SystemTime.Now() - LocalPreferencesHelper.LastDateTimeTermsNotificationWasShown).Days > 0)
```

Comment:
Maybe it would be better SystemTime.Now().Date > LocalPreferencesHelper.LastDateTimeTermsNotificationWasShown).Date?

Reply:
Not really, because it won't cover 24 hour period. E.g., 01.01.2021 23.59 and 02.01.2021 00.01 will give us true already and trigger a notification then.

Reply:
ok, makes sense :)